src/asset_folder_importer/tests/test_importer_thread.py

```diff
@@ -235,8 +234,8 @@ def test_md_render(self):
 
         mock_db = MagicMock(target=importer_db)
         mock_fileref = MagicMock(target=VSFile)
-        mock_fileref.mtime = datetime(2018,01,04,15,32,00)
-        mock_fileref.ctime = datetime(2018,01,04,15,30,00)
+        mock_fileref.mtime = datetime(2018,0o1,0o4,15,32,00)
```

Comment:
again surely just `1` and`4` rather than converting them to octal?

Reply:
Changed.